src/test/java/com/EighthLight/app/TestConfig.java

```diff
@@ -18,20 +18,11 @@
     private ArrayList<String> gameModeIncorrectInput;
     private ArrayList<String> boardSizeIncorrectInput;
     private ArrayList<String> playerOrderingTwoUserInputs;
-
-    @BeforeEach
-    public void setUp() {
-        defaultUserInputs = new ArrayList<>(Arrays.asList("X", "O", "1", "1", "1", "3"));
-        duplicateSymbolUserInputs = new ArrayList<>(Arrays.asList("X", "X", "O", "1", "1", "3"));
-        gameModeOneUserInputs = new ArrayList<>(Arrays.asList("X", "O", "1", "1","3"));
-        gameModeTwoUserInputs = new ArrayList<>(Arrays.asList("X", "O", "2", "1", "3"));
-        playerOrderingTwoUserInputs = new ArrayList<>(Arrays.asList("X", "O", "2", "2", "3"));
-        gameModeIncorrectInput = new ArrayList<>(Arrays.asList("X", "O", "5", "1", "1", "3"));
-        boardSizeIncorrectInput = new ArrayList<>(Arrays.asList("X", "O", "1", "", "Ten", "3"));
-    }
```

Comment:
Removed to improve readability. Each input ArrayList is now a local variable for its specific test.